aoc-client/src/lib.rs

```diff
@@ -421,6 +421,77 @@ impl AocClient {
         Ok(())
     }
 
+    fn get_global_leaderboard_html(&self) -> AocResult<String> {
+        debug!("🦌 Fetching global leaderboard for {}", self.year);
+
+        let url = format!("https://adventofcode.com/{}/leaderboard", self.year);
+        let response = reqwest::blocking::get(url)?;
```

Comment:
Any reason to use `reqwest` directly instead of calling `http_client`? This way you're missing some common error handling there and also not passing the user agent header.

Reply:
I just used ``reqwest`` directly because I saw the session cookie wasn't required for grabbing the global leaderboard. Happy to switch it over for the error handling :)